cli/remote_build.go

```diff
@@ -780,9 +782,15 @@ func (cmd *RemoteCmd) submitBuild(project *Project, tagMap map[string]string) er
 	case 401:
 		submitErrored()
 		return fmt.Errorf("Unauthorized, authentication failed.\nPlease `yb login` again.")
-	case 400:
+	case 403:
 		submitErrored()
-		return fmt.Errorf("Invalid data sent to the YB API")
+		if cmd.publicRepo {
+			return fmt.Errorf("This should not happen, please open a support inquery with YB")
+		} else {
+			return fmt.Errorf("Tried to build a private repository of a organization of which you're not part of.\nPlease check with the '%s' admins", project.OrgSlug)
```

Comment:
I would remove the second sentence so we're not revealing organization details (even slug) with an unauthorized user.